docs/source/schema_cancer.rst

```diff
@@ -50,15 +50,15 @@ An example is shown below:
 
 They are as follows:
 
-1. ``patientName`` -- name of the patient, used for identifying the patient in the sample sheet.
-2. ``sampleName`` -- name of the sample, used for identifying the sample for the patient in the sample sheet (the combination of patient and sample must be unique in the sheet).
+1. ``patientName`` -- name of the patient, used for identifying the patient in the sample sheet as ``BioEntity`` in :ref:`high_level_overview`
+2. ``sampleName`` -- name of the sample, used for identifying the sample for the patient in the sample sheet (the combination of patient and sample must be unique in the sheet). sampleName translates to ``BioSample`` in :ref:`high_level_overview`.
```

Comment:
Each sentence should start on a new line, ``sampleName`` should be teh formatting. Also note that "translate" is the wrong word here IMO.  Rather, it should be "is assigned to the ``secondaryId`` of the ``BioSample``. Use double-backticks for ``BioEntity``, ``BioSample``, etc. consistently in your changes please.